<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="forwardButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backwardButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectToRoverButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QComboBox" name="avail_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="avail_label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>264</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Joystick:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="crawlerConnectedLabel">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>15</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>disconnected</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
